src/test/java/nl/tudelft/ewi/devhub/server/backend/Bootstrapper.java

```diff
@@ -146,6 +151,9 @@ public void prepare(String path) throws IOException, ApiError, GitClientExceptio
                 assignmentEntity.setAssignmentId(assignment.getId());
                 assignments.persist(assignmentEntity);
                 log.debug("Persistted assignment {} in {}", assignmentEntity.getName(), course.getCode());
+                
+                // Store for later use to insert deliveries
+                lastAssignment = assignmentEntity;
```

Comment:
Could you store the assignments in a `Map<Long, Assignment>` and retrieve the assignments from there. You'll see where this comes in handy in the next comment..